### PR TITLE
Create layout for empty state

### PR DESCRIPTION
**Pull request by CtrlSpice**
Basically just figuring out how to make it look nice. Nothing is functional yet, but this is the part that takes me the longest. Also, a first peak at Lulu the axolotl, in sample form. 😍 
empty-state-day
emty-state-night

### CODE REVIEW

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      width="100%"
+      overflowY="scroll"
+    >
+      <Alert
+        status="info"
+        variant="solid"
+        minHeight="30px"
+      >
+        <AlertIcon />
+        No data yet. Checking again in 15 seconds...
```
Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:34]: 15 seconds is probably too slow given reloads are so quick. Maybe drop this to 5?

Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:34]:
```suggestion
        No data yet. Refreshing in 5 seconds...
```

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              </OrderedList>
+            </Box>
+            <Text>
+              Alternately, you can load some example data to get a feel for the
+              tool.
+            </Text>
+          </Stack>
+        </CardBody>
+        <CardFooter>
+          <Button colorScheme="pink">Load Example Data</Button>
```
Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:75]:
```suggestion
          <Button colorScheme="pink">Load Sample Data</Button>
```

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-01-31 16 20 05

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        align="center"
+        width="50%"
+        maxWidth="700px"
+        margin="64px"
+        variant="filled"
+      >
+        <CardHeader>
+          <Image
+            src="assets/images/lulu.jpg"
+            alt="An axolotl named Lulu, striking a heroic pose while gazing at a field of stars through a telescope"
```
[the remark this answers is not in the document]
Comment (CtrlSpice) [reply desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:46]: Lulu Alxol'Otel, the first of her name, valiant adventurer, observability queen, pink AF. ❤️

**Pull request merged**